### PR TITLE
Add `Tux.set_velocity()` to scripting API.

### PR DESCRIPTION
**Pull request by tylerandari13**
Pretty self explanatory.

We have Tux.get_velocity_x() and Tux.get_velocity_y() so why not add a velocity setter as well?

### CODE REVIEW

**Review by mrkubax10 — APPROVED** [review]

File: src/scripting/player.hpp [hunk 1]
```diff
@@ -193,6 +193,11 @@ class Player final
    */
   float get_velocity_y() const;
 
+  /**
+   * Sets the velocity of the player to a programmable/variable speed.
+   */
```
Comment (tobbi) [inline src/scripting/player.hpp:198]: Add documentation for params.

Comment (tylerandari13) [reply src/scripting/player.hpp:198]: I noticed `Tux.set_pos()` doesn't have params either, so to keep things consistent i didn't add params here either.

Comment (mrkubax10) [reply src/scripting/player.hpp:198]: It should have param documentation for scripting reference.

Comment (tylerandari13) [reply src/scripting/player.hpp:198]: ok then, ill add params to the function

**Pull request merged**